src/runtime/entries/deno-server.ts

```diff
@@ -0,0 +1,44 @@
+// @ts-ignore
+import process from 'https://deno.land/std/node/process.ts'
+// @ts-ignore
+import { serve, serveTls } from 'https://deno.land/std/http/server.ts'
+
+import destr from 'destr'
+
+import { handler } from './deno'
+import { useRuntimeConfig } from '#internal/nitro'
+
+const cert = process.env.NITRO_SSL_CERT
+const key = process.env.NITRO_SSL_KEY
+const port = destr(process.env.NITRO_PORT || process.env.PORT) || 3e3
+const hostname = process.env.NITRO_HOST || process.env.HOST
+
+if (cert && key) {
```

Comment:
Is the idea is to get this file being written in "idomatic Deno"? If so I can surely make some suggestions.

Is there a reason to import `process` polyfill other than registering `unhandledRejection` and `uncaughtException` events? Does any of the imported code depend on it?

Reply:
Many of the Vue dependencies rely on `process.env.NODE_ENV` to determine whether to return prod/dev code. This of course needs to be updated there to use production/development import conditions but for now it's what we're working with.

Reply:
We also provide `import.meta.env` via our own polyfill from `process.env`